src/annotations/data_preparation.py

```diff
@@ -88,16 +92,22 @@ def extract_dataframe_pixels_values_from_tif_and_polygons(
     @name_tif_file: name of the tif_dataset object, so it can be matched with the correct row from polygon_df (polygon_gdf["name"])
     @return pandas DataFrame with a pixel per row
     """
-    polygon_gdf = polygon_gdf
 
     dfs = []
-    for _, row in polygon_gdf.iterrows():
-        if row["name"] == name_tif_file:
+    for aindex, row in polygon_gdf.iterrows():
+        if row["name"] == name_annotations:
```

Comment:
I'm not really sure what you're trying to achieve with this name_annotations functionality. From how I read this now, this 'extract_dataframe_pixels_values_from_tif_and_polygons' will now only work if you also pass it the name of the file. Is that the idea? That will probably break the code I wrote in 'transform_polygon_annotations.ipynb' for one thing.

Reply:
The gpkg is more like a database, were the name of table is a reference to where the annotations are made.
But there are also global tables which have annotations of multiple tif files.

As such I want to make disctintion between the table name which is name_annotations here and the name of the tif file. As such the name_annotations variable was born.